File: web-client/src/app/interfaces/chat-interfaces/message.ts
import { MessageStatus } from "src/app/enums/message-status";
import { UserModel } from "../user-interfaces/user-model";
import { Chat } from "./chat";

export interface Message {
    id: string
    text?: string,
    chatId?: string,
    chat?: Chat,
    userId: string,
    user: UserModel,
    createdAt?: Date;
    status?: MessageStatus;
}

File: TripsWebApp.DAL/Entities/ChatEntities/Message.cs
namespace BlaBlaCar.DAL.Entities.ChatEntities
{
    public class Message:BaseEntity
    {
        public string Text { get; set; }
        public Guid ChatId { get; set; }
        public Chat Chat { get; set; }
        public Guid UserId { get; set; }
        public ApplicationUser User { get; set; }
    }
}
